.gitignore

```diff
@@ -16,4 +16,5 @@ npm-debug.log
 /.nyc_output
 
 # source
-dist
\ No newline at end of file
+dist
+example
```

Comment:
I have on my local repository an example dir which houses a sample nest application for easy testing. Not sure if it conflicts with anyone